### ISSUE TITLE
关于 在新闻预料上 做文本分类模型的问题

### ISSUE THREAD

**Issue by fanruifeng** [yaleimeng/TextClassify_with_BERT#2]
你好, 作者 , 我 基于Bert 模型 用 开源的新闻语料 来预测, 发现 最后在验证集下 出来的loss 很高 在6-8之间, 无论怎么调试 参数, 都在这之间, 我INFO:tensorflow:evaluation_loop marked as finished
INFO:tensorflow:***** Eval results *****
INFO:tensorflow:  eval_accuracy = 0.10002001
INFO:tensorflow:  eval_f1 = 0.9474681
INFO:tensorflow:  eval_loss = 6.468474
INFO:tensorflow:  eval_precision = 0.90018004
INFO:tensorflow:  eval_recall = 1.0
INFO:tensorflow:  global_step = 4687
INFO:tensorflow:  loss = 6.4671826
不知道什么原因造成的, 我猜想应该是 数据预处理部分,   希望大佬帮我解惑, 如果 方便的话, 加个联系方式  再次感谢大佬, 我的联系方式: qq：1125729232
